roles/pulp_common/templates/pulpcore.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Pulpcore Application
```

Comment:
Perhaps a better description to explain that this service manages all the other services?

Reply:
do you have a specific suggestion? Naming is pretty hard!